README.md

```diff
@@ -64,53 +66,56 @@ flowchart TB
 
 ## Tests
 
-- Performance tests on desktop and mobile devices
+### Performance tests on desktop and mobile devices
   
-  [Lighthouse](https://github.com/GoogleChrome/lighthouse) is a tool that analyzes web apps and web pages to collect performance metrics and insights on developer best practices. To avoid significant performance drops introduced by Pull Requests, a pair of GitHub actions using [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci) are running against PRs to collect the performance metrics of the code with the proposed changes at desktop and mobile devices (a report containing the results is hosted on a URL that is availaible at the end of the actions log).
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) is a tool that analyzes web apps and web pages to collect performance metrics and insights on developer best practices. To avoid significant performance drops introduced by Pull Requests, a pair of GitHub actions using [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci) are running against PRs to collect the performance metrics of the code with the proposed changes at desktop and mobile devices (a report containing the results is hosted on a URL that is availaible at the end of the actions log).
```

Comment:
I couldn't find any evidence that we are in fact running these tests on open PRs. And then I went to check the [repo's actions](https://github.com/vtexdocs/helpcenter/actions) and saw that Lighthouse actions are disabled.

Reply:
Great! I will remove it.